src/MSBuild/Resources/xlf/Strings.pt-BR.xlf

```diff
@@ -970,7 +970,7 @@ isoladamente.
       </trans-unit>
       <trans-unit id="InvalidGraphBuildValue">
         <source>MSBUILD : error MSB1057: Graph build value is not valid.</source>
-        <target state="translated">MSBUILD : error MSB1057: o valor de build do grafo não é válido.</target>
+        <target state="translated">MSBUILD : error MSB1057: O valor de build do gráfico não é válido. {0}</target>
```

Comment:
@cristianosuzuki77 Translator introduced a parameter here that doesn't exist in the original string which caused a build break.

```suggestion
        <target state="translated">MSBUILD : error MSB1057: O valor de build do gráfico não é válido.</target>
```